autogtp/main.cpp

```diff
@@ -88,6 +89,19 @@ int main(int argc, char *argv[]) {
     QTextStream cerr(&caFile);
 #else
     QTextStream cerr(stderr, QIODevice::WriteOnly);
+#endif
+#ifdef WIN32
+	//We need to make sure these files we need are there before calling them. Otherwise it will result in nullptr.
+	QFileInfo curl_exe("curl.exe");
+	QFileInfo gzip_exe("gzip.exe");
+	QFileInfo leelaz_exe("leelaz.exe");
+	if (!(curl_exe.exists() && gzip_exe.exists() && leelaz_exe.exists()))
+	{
+		auto t = curl_exe.exists();
+		cerr << "exe files are missing!"<< endl;
```

Comment:
Should probably have a cleaner error message. (And a note "Press enter to continue or something"?)